otter/convergence/steps.py

```diff
@@ -73,6 +80,12 @@ def set_server_name(server_config_args, name_suffix):
     return set_in(server_config_args, ('server', 'name'), name)
 
 
+def append_stack_uuid(stack_config, uuid):
```

Comment:
If you want to just inline the body of this function into its only call-site I think that'd be fine, since it's so simple. But no big deal either way.